fixtures/u2_type_method_private.go

```diff
@@ -0,0 +1,7 @@
+package fixtures
+
+type Foo int
```

Comment:
There is some inconsistency because in some test filenames the types are called `u2_class`, but here are `called u2_type`, is there a rationale for this?